src/main/java/org/folio/inventory/services/MoveItemIntoStatusService.java

```diff
@@ -57,14 +61,38 @@ public MoveItemIntoStatusService(ItemCollection itemCollection, Clients clients)
       .thenCompose(itemCollection::update);
   }
 
+  public CompletableFuture<Item> processMarkItem(WebContext context,ItemStatusName statusName) {
```

Comment:
Given that we have settled on the language of `mark` to refer to a status transition rather than `move`, I think it might make sense to rename this method (and class).

What do you think to `markItemStatusAs`?

```suggestion
  public CompletableFuture<Item> markItemStatusAs(WebContext context, ItemStatusName statusName) {
```